helm_deploy/apply-for-legal-aid/templates/deployment_web.yaml

```diff
@@ -31,7 +31,6 @@ spec:
 {{ toYaml .Values.clamav.resources | indent 12 }}
         - name: web
           image: '{{ .Values.image.repository }}:{{ .Values.image.tag }}'
-          imagePullPolicy: IfNotPresent
```

Comment:
`imagePullPolicy`'s default value is `IfNotPresent`.
The effect of omitting it is:
- if the image's tag is `latest`, force a pull
- otherwise, use an existing image if it's there.

See https://kubernetes.io/docs/concepts/containers/images/#updating-images :

Reply:
Without this change, the container would not be an updated when re-deploying new code with the tag `latest`.
It's strange because it worked in `live-0` but it doesn't in `live-1`.
Maybe it's a different cluster setting or a different version

Reply:
Actually, I'm removing this change. Somehow, I was under the impression that we used `latest` when deploying to staging/prod. But we are always using the git commit ID. So the previous code worked